opba-db/src/main/resources/migration/migrations/0002-mock-data.xml

```diff
@@ -42,7 +42,7 @@
         </update>
     </changeSet>
 
-    <changeSet id="load_csv_banks_random" author="mhr">
+    <changeSet id="load_csv_banks_random" author="mhr" context="search-performance-test">
```

Comment:
You need to ensure that `TestBankSearchPerformance` uses Liquibase context `search-performance-test`

Reply:
yes i've tested it already. With and without i could check the database and see if the test bank are present or not.